docs/json-rpc-api.md

```diff
@@ -103,6 +103,9 @@ Here are the currently supported procedures, the arguments they require, and the
 `filtertype([string type])`
  * Set the filter for the given type.
 
+ `filterrating([string type])`
```

Comment:
Please add a note it only works if yts api is used as custom movie api... so people making the remote apps will know and not waste time thinking it is broken.

Reply:
should I remove this part all together? not sure if it's needed at all, added to PR for a discussion

Reply:
No. I think we kept it but imo it need a text in description to inform people that it will only works with yts api so they will not think it's bugged.


```
 `filterrating([string rating])`
  * /!\ Caution /!\ - Only works if client movie api is set to use YTS api
  * Set the filter for the given minimum rating.

```